models/todo.js

```diff
@@ -1,19 +1,49 @@
 const mongoose = require("mongoose");
 
-//sample for testing
-const todoSchema = mongoose.Schema({
-  title: {
-    type: String,
-    required: true,
+const todoSchema = mongoose.Schema(
+  {
+    matches: [
+      {
+        matchDate: {
+          type: String,
+          required: true,
+        },
+        league: {
+          type: String,
+          required: true,
+        },
+        homeTeam: {
+          type: String,
+          required: true,
+        },
+        awayTeam: {
+          type: String,
+          required: true,
+        },
+        homeScore: {
+          type: String,
+          required: true,
+        },
+        awayScore: {
+          type: String,
+          required: true,
+        },
+        matchStatus: {
+          type: String,
+          required: true,
+        },
+        homeLogo: {
+          type: String,
+          required: true,
+        },
+        awayLogo: {
+          type: String,
+          required: true,
+        },
+      },
+    ],
   },
-  content: {
-    type: String,
-    required: true,
-  },
-  date: {
-    type: String,
-    required: true,
-  },
-});
+  { collection: "sampleMatchData"}
+);
 
-module.exports = mongoose.model("Todo", todoSchema);
+module.exports = mongoose.model("Todo", todoSchema);
```

Comment:
do not reuse todo, create your own models with the correct variable and file name